### ISSUE TITLE
[2.1.1] Not compiling

### ISSUE THREAD

**Issue by clayne11** [nathantreid/meteor-css-modules#62]
I'm using SCSS and Meteor `1.3.5.1`. Everything works fine in `2.1.0`. Updating to `2.1.1` and beyond causes the following:

```
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13:
   Unexpected token (2:14)
   at Parser.pp.raise
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13)
   at Parser.pp.unexpected
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:89:8)
   at Parser.pp.semicolon
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:76:38)
   at Parser.pp.parseImport
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:1007:8)
   at Parser.pp.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:141:56)
   at Parser.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/plugins/flow.js:30:22)
   at Parser.pp.parseBlockBody
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:529:21)
   at Parser.pp.parseTopLevel
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:36:8)
   at Parser.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/index.js:129:19)
   at Object.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/index.js:47:47)
   at parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:38:29)
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:58:15
   at Cache.Cp.get
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/cache.js:94:19)
   at Object.compile
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:47:23)
   at Object.Babel.compile (packages/babel-compiler/babel.js:26:1)
   at CssModulesBuildPlugin.addCompileResult (packages/mss/css-modules-build-plugin.js:294:17)
   at packages/mss/css-modules-build-plugin.js:252:12
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:32:39


   While processing files with nathantreid:css-modules (for target os.osx.x86_64):

   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13:
   Unexpected token (2:14)
   at Parser.pp.raise
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13)
   at Parser.pp.unexpected
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:89:8)
   at Parser.pp.semicolon
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:76:38)
   at Parser.pp.parseImport
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:1007:8)
   at Parser.pp.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:141:56)
   at Parser.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/plugins/flow.js:30:22)
   at Parser.pp.parseBlockBody
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:529:21)
   at Parser.pp.parseTopLevel
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:36:8)
   at Parser.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/index.js:129:19)
   at Object.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/index.js:47:47)
   at parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:38:29)
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:58:15
   at Cache.Cp.get
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/cache.js:94:19)
   at Object.compile
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:47:23)
   at Object.Babel.compile (packages/babel-compiler/babel.js:26:1)
   at CssModulesBuildPlugin.addCompileResult (packages/mss/css-modules-build-plugin.js:294:17)
   at packages/mss/css-modules-build-plugin.js:252:12
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.3.h3isao++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:32:39
```

**Comment by akanix42** [comment]
Can you update to `nathantreid:css-modules@2.1.4-alpha.4` and paste in the new error? I added exception logging for the Babel compiler. The output should be similar to this:

```
   Processing files with nathantreid:css...
 /~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unable to compile node_modules/react-toolbox/lib/animations/slide-left.scss: SyntaxError: Unexpected token (5:28)
Input:
// <start of file>
 <
const styles = {"enter":"_node_modules_react_toolbox_lib_animations__slide_left__enter","leave":"_node_modules_react_toolbox_lib_animations__slide_left__leave","enterActive":"_node_modules_react_toolbox_lib_animations__slide_left__enterActive","leaveActive":"_node
_modules_react_toolbox_lib_animations__slide_left__leaveActive"};
export { styles as default, styles };
// <end of file>

/~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
=> Errors prevented startup:

   While processing files with nathantreid:css-modules (for target web.browser):
```

**Comment by akanix42** [comment]
Make sure to use `nathantreid:css-modules@2.1.4-alpha.4` - I initially said to use alpha.3, but I accidentally published that with the invalid character I was using to test the Babel error. :grimacing:

**Comment by clayne11** [comment]
```
=> Started proxy.
=> Started MongoDB.
=> Meteor 1.4 is available. Update this project with 'meteor update'.

Changes to your project's package version selections:

nathantreid:css-modules                      upgraded from 2.1.0 to 2.1.4-alpha.4
nathantreid:css-modules-import-path-helpers  upgraded from 0.1.3 to 0.1.4

=> Starting gadicc:hot-build Accelerator (PH9) on port 3002.

   Processing files with nathantreid:css...  -
/~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unable to compile imports/ui/components/redux-form-components/components/withBuiltInTitle/styles.scss: SyntaxError: Unexpected token (2:19)
Source:
// <start of file>
 import ''../../form-component-mixins.scss'';
import modules from 'meteor/modules';
modules.addStyles("._imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__root {\n  margin-top: 14px; }\n\n._imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helperWrapper {\n  padding-top: 8px; }\n\n._imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helper {\n  color: rgba(0, 0, 0, 0.38); }\n\n._imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__error {\n  color: #ff6e40; }\n/*# sourceMappingURL=styles.scss.map */");
const styles = {"title-color":"rgba(0, 0, 0, 0.87)","active-title-font-size":"16px","root":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__root","helperWrapper":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helperWrapper","helper":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helper _imports_ui_components_redux_form_components__form_component_mixins__helper _imports_ui_components_redux_form_components__form_component_mixins__helperBase","error":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__error _imports_ui_components_redux_form_components__form_component_mixins__helperBase"};
export { styles as default, styles };
// <end of file>

/~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   Processing files with nathantreid:css...  /
/~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unable to compile imports/ui/components/redux-form-components/components/withBuiltInTitle/styles.scss: SyntaxError: Unexpected token (2:19)
Source:
// <start of file>
 import ''../../form-component-mixins.scss'';
const styles = {"title-color":"rgba(0, 0, 0, 0.87)","active-title-font-size":"16px","root":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__root","helperWrapper":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helperWrapper","helper":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__helper _imports_ui_components_redux_form_components__form_component_mixins__helper _imports_ui_components_redux_form_components__form_component_mixins__helperBase","error":"_imports_ui_components_redux_form_components_components_withBuiltInTitle__styles__error _imports_ui_components_redux_form_components__form_component_mixins__helperBase"};
export { styles as default, styles };
// <end of file>

/~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
=> Errors prevented startup:

   While processing files with nathantreid:css-modules (for target web.browser):

   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13:
   Unexpected token (2:19)
   at Parser.pp.raise
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13)
   at Parser.pp.unexpected
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:89:8)
   at Parser.pp.semicolon
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:76:38)
   at Parser.pp.parseImport
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:1007:8)
   at Parser.pp.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:141:56)
   at Parser.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/plugins/flow.js:30:22)
   at Parser.pp.parseBlockBody
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:529:21)
   at Parser.pp.parseTopLevel
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:36:8)
   at Parser.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/index.js:129:19)
   at Object.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/index.js:47:47)
   at parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:38:29)
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:58:15
   at Cache.Cp.get
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/cache.js:94:19)
   at Object.compile
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:47:23)
   at Object.Babel.compile (packages/babel-compiler/babel.js:26:1)
   at tryBabelCompile (packages/mss/css-modules-build-plugin.js:306:18)
   at CssModulesBuildPlugin.addCompileResult (packages/mss/css-modules-build-plugin.js:293:11)
   at packages/mss/css-modules-build-plugin.js:251:12
   at
   /Users/clayne/.meteor/packages/less/.2.6.5.1wcmgw0++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:32:39


   While processing files with nathantreid:css-modules (for target os.osx.x86_64):

   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13:
   Unexpected token (2:19)
   at Parser.pp.raise
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/location.js:22:13)
   at Parser.pp.unexpected
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:89:8)
   at Parser.pp.semicolon
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/util.js:76:38)
   at Parser.pp.parseImport
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:1007:8)
   at Parser.pp.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:141:56)
   at Parser.parseStatement
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/plugins/flow.js:30:22)
   at Parser.pp.parseBlockBody
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:529:21)
   at Parser.pp.parseTopLevel
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/statement.js:36:8)
   at Parser.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/parser/index.js:129:19)
   at Object.parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/babylon/lib/index.js:47:47)
   at parse
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:38:29)
   at
   /Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:58:15
   at Cache.Cp.get
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/cache.js:94:19)
   at Object.compile
   (/Users/clayne/.meteor/packages/nathantreid_css-modules/.2.1.4-alpha.4.r0w38o++os+web.browser+web.cordova/plugin.mss.os/npm/node_modules/meteor/babel-compiler/node_modules/meteor-babel/index.js:47:23)
   at Object.Babel.compile (packages/babel-compiler/babel.js:26:1)
   at tryBabelCompile (packages/mss/css-modules-build-plugin.js:306:18)
   at CssModulesBuildPlugin.addCompileResult (packages/mss/css-modules-build-plugin.js:293:11)
   at packages/mss/css-modules-build-plugin.js:251:12
   at
   /Users/clayne/.meteor/packages/less/.2.6.5.1wcmgw0++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:32:39


=> Your application has errors. Waiting for file change.
```

**Comment by clayne11** [comment]
It seems to be unhappy with my use of `@value` I believe.

**Comment by akanix42** [comment]
Can you post the contents of `imports/ui/components/redux-form-components/components/withBuiltInTitle/styles.scss`?
It seems to be importing `../../form-component-mixins.scss` and as you may have noticed in the error output, the import path is for some reason surrounded with double single quotes: `import ''../../form-component-mixins.scss'';`

I'm not sure why that's happening; I _think_ I see how to fix it, but I'd like to reproduce it first.  ;)

**Comment by akanix42** [comment]
I went ahead and made a release for this in case its that easy. Can you try this with `nathantreid:css-modules@2.2.2-issue62.1`?

**Comment by clayne11** [comment]
That worked, thanks.

**Comment by akanix42** [comment]
Awesome! This is available in v2.2.2.
